lib/components/dropdown.vue

```diff
@@ -28,7 +28,7 @@
             <span class="sr-only">{{toggleText}}</span>
         </b-button>
 
-        <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
+        <div :class="['dropdown-menu',{'dropdown-menu-right': right, show: visible}]"
```

Comment:
This was just BS-Beta behavior that sets both. I don't think is required, but being consistent with original library is a good idea :)